main.go

```diff
@@ -52,10 +53,17 @@ import (
 //go:generate rsrc -arch 386 -manifest resources/artifactcollector32.exe.user.manifest -ico resources/artifactcollector.ico -o resources/artifactcollector32.user.syso
 
 func main() {
+	outDir := flag.String("o", "./", "Output directory for forensic store and log file")
+	flag.Parse()
+
 	var artifacts []goartifacts.ArtifactDefinition
 	artifacts = append(artifacts, artifactsgo.Artifacts...)
 	artifacts = append(artifacts, assets.Artifacts...)
-	collection := run.Run(assets.Config, artifacts, assets.FS)
+
+	config := assets.Config
+	config.OutputDir = *outDir
```

Comment:
If an output directory is set in the config, but the flag is not set we should not overwrite it.

Reply:
Ah, thanks. I'll fix it.

Reply:
But as a second thought: what should user expect, when he specifies output directory in config and as a flag. The flag seems easier to change, so a user could think that it should override the config value. What you think?

Reply:
I agree. The flag should overwrite the config. And the config should overwrite the default value.

Reply:
Please take a look at the 3fcae7f.

Reply:
Looks good to me.